tests/testthat/_snaps/signal.md

```diff
@@ -9,3 +9,16 @@
     Output
       <lifecycle_stage: foo(arg) is superseded>
 
+# signal_stage supports with
+
+    Code
+      (expect_condition(signal_stage("superseded", "foo()", "bar()")))
+    Output
+      <lifecycle_stage: foo() is superseded
```

Comment:
Should we add a `testthat_print()` method for bare conditions to make the display less awkward? hmm I guess that's not really worth the effort.

Reply:
Yeah, and it can also be resolved by #91, which would give other folks an easy way to see these annotations.